docs/source/benchmarking.rst

```diff
@@ -127,11 +77,92 @@ Example for the 78Mbp simulated community
 
-    Unfortunately, at the moment ``gdown`` doesn't support downloading entire directories from Google drive.
+    2. Unfortunately, at the moment ``gdown`` doesn't support downloading entire directories from Google drive.
     There is an open `Pull request <https://github.com/wkentaro/gdown/pull/90#issue-569060398>`_ on the ``gdown`` repository
     addressing this specific issue which we are keeping a close eye on and will update this documentation when it is merged.
 
+Benchmark clustering
+^^^^^^^^^^^^^^^^^^^^
+
+Here we are benchmarking the clustering step. All the simulated communities are being benchmarked. You don't have to run all of them, this is just for tutorial purposes.
+
+.. code:: bash
+
+    for community_size in ${community_sizes[@]};do
+        autometa-benchmark \
+            --benchmark clustering \
+            --predictions simulated/${community_size}/binning.tsv.gz \
+            --reference simulated/${community_size}/reference_assignments.tsv.gz \
+            --output-wide ${community_size}.clustering_benchmarks.wide.tsv.gz \
+            --output-long ${community_size}.clustering_benchmarks.long.tsv.gz
+    done
+
+Benchmark classification
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously we benchmarked the clustering step. Now we are going to benchmark the taxon profiling step. The only difference is that ``--classification`` flag is provided instead of ``--clustering`` flag.
+
+.. code:: bash
+
+    autometa-benchmark \
+        --benchmark classification \
+        --predictions $taxonomy ${community_size}.taxonomy.tsv.gz \
+        --reference simulated/${community_size}/reference_assignments.tsv.gz \
+        --output-wide ${community_size}.classification_benchmarks.wide.tsv.gz \
+        --output-classification-reports taxa_reports \
+        --ncbi /ncbi
+
+Benchmark clustering-classification
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This will benchmark the taxonomic assignemnt of each bin. To run this you'll need ``binning.tsv.gz`` file for the specific community. This can be downloa by running:
+
+.. code:: bash
+    autometa-download-dataset \
+        --community-type simulated \
+        --community-sizes 78Mbp \
+        --file-names binning.tsv.gz\
+        --dir-path simulated
+
+Now running the command to benchmark:
+
+.. code:: bash
+
+    autometa-benchmark \
+        --benchmark binning-classification \
+        --predictions 78Mbp/binning.tsv \
+        --reference 78Mbp/reference_assignments.tsv.gz \
+        --output-wide binning_classification_wide.tsv.gz
+
+Specifying mulile results
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``autometa-benchmark`` provides the flexibility of specifying multiple output files of the same dataset and benchmarking them simultaneously. For example, in case you have done the clustering for a sample using three different binner, you can benchmark all of them simulateously. This is really handy when benchmarking multiple tools on the same dataset.
```

Comment:
❗  Could you place this with the autometa-clustering example in the tutorial and add it under a note?❗ 

```suggestion

.. note::

    ``autometa-benchmark`` provides the flexibility of benchmarking multiple results 
    corresponding to the same dataset. For example, if you have clustering results for 
    a sample using three different binners, you can benchmark all of them, simultaneously. 
    This is really handy when benchmarking multiple tools on the same dataset.
```